src/components/CohortList/CohortListStyle.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import styled from "styled-components";
+
+export const List = styled.ul`
```

Comment:
General question: do we want to call the styled components that we are exporting with "Style" at the end or not? @vindhya @martapa @yuliwna1

Reply:
I agree that the naming should be consistent if it's a component being used purely for style. I'm good with the "Style" suffix at the end.

After doing a little googling, I've seen people using the prefix "Styled" for the styled-components, like "StyledList". Also seen people using "Wrapper" as a suffix like "ListWrapper".